packages/assets-controllers/src/TokenBalancesController.test.ts

```diff
@@ -14,6 +14,13 @@ import {
   TokenBalancesController,
 } from './TokenBalancesController';
 
+const TokensControllerMessenger = new ControllerMessenger<
```

Comment:
I was a bit shocked as well to see that a `TokensController` was instantiated for unit tests on all those other controllers.

I'll have a look and see how much of a pain would be to just mock them, because as I mentioned in a previous comment doing so is not very sustainable (and it makes these integration tests, not unit tests).